Microsoft.AspNet.AngularServices/project.json

```diff
@@ -1,20 +1,16 @@
 {
   "version": "1.0.0-alpha7",
-  "description": "Microsoft.AspNet.AngularServices Class Library",
-  "authors": [
-    "Microsoft"
-  ],
-  "tags": [
-    ""
-  ],
-  "projectUrl": "",
-  "licenseUrl": "",
+  "description": "ASP.NET 5 server-side Angular services.",
```

Comment:
It also contains client-side stuff. Maybe rephrase to "Helpers for building Angular 2 applications on ASP.NET 5"